src/simplnx/Filter/Actions/DeleteDataAction.cpp

```diff
@@ -112,7 +112,7 @@ DeleteDataAction::~DeleteDataAction() noexcept = default;
 
 Result<> DeleteDataAction::apply(DataStructure& dataStructure, Mode mode) const
 {
-  auto* targetObject = dataStructure.getDataAs<BaseGroup>(path());
+  // auto* targetObject = dataStructure.getDataAs<BaseGroup>(path());
```

Comment:
Should we just delete this line or are we leaving it for some historical context

Reply:
The commented out lines below referenced it so I also commented out this one. I don't have the context for those lines being commented out.